### PR TITLE
Upgrade Sentry implementation

### PR DESCRIPTION
**Pull request by sallar**
This PR does the following:

+ Upgrades Raven dependency to the latest
+ Changes Sentry config to point to the new account

### CODE REVIEW

File: app/src/common/reporter.js [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
 import isDev from 'electron-is-dev';
 import unhandled from 'electron-unhandled';
 
-let ravenClient;
+let Raven;
 
 function init() {
   if (!isDev) {
-    const raven = require('raven');
+    Raven = require('raven');
```
Comment (albinekb) [inline app/src/common/reporter.js]: Hmm, should we use `raven-js` in the renderer, and `raven` in the main process? 🤔

File: app/package.json [hunk 2]
```diff
@@ -39,7 +39,8 @@
     "ms": "^2.0.0",
     "npm": "^4.6.1",
     "object-path": "^0.11.3",
-    "raven": "^0.12.1",
+    "raven": "^2.3.0",
```
Comment (albinekb) [inline app/package.json]: now we depend on both these, do we need both `raven` and `raven-js`? Afaik one is for the renderer (browser side) and the other for the main (node) side?

Comment (sallar) [reply app/package.json]: Oh.. I was wondering about that. Let me double check (both worked). Checking now

Comment (albinekb) [reply app/package.json]: I **think*

Comment (albinekb) [reply app/package.json]: I **think** it's important to use both of them as they are tailored for their respective environments, like tracking clicked links in the DOM for the browser one while the node one is tracking more OS specific stuff.

Comment (sallar) [reply app/package.json]: @albinekb I agree. Done :)

**Comment by sindresorhus** [comment]
Tested it and it didn't work. Apparently, the DSN needs to be different in the main process version.

**Comment by sindresorhus** [comment]
With https://github.com/wulkano/kap/pull/307/commits/879c6aa1bc3af586e3955f7129d8d89220daef4b, it works perfectly for both main and renderer.

**Pull request merged**